osu.Game/Screens/Play/GameplayMenuOverlay.cs

```diff
@@ -270,5 +272,8 @@ protected override bool Handle(UIEvent e)
 
             return base.Handle(e);
         }
+
+        // Locally allow samples to playback without considering gameplay pause state.
+        public IBindable<bool> SamplePlaybackDisabled { get; } = new Bindable<bool>();
```

Comment:
Rather than do this, my primary question would be: if `PauseOverlay` does not want to respect playback disable, then why is it using the class that *supports* playback disable? Why is it not using `SkinnableSample` singular?